Formula/virtualpg.rb

```diff
@@ -21,6 +21,7 @@ class Virtualpg < Formula
   end
 
   depends_on "libspatialite"
+  depends_on "postgresql@14"
```

Comment:
This was added because if it's referenced in `configure` directly is should be present here rather than relying on the `postgis` indirect dependency.